website/content/prototyping/_index.markdown

```diff
@@ -1,195 +1,17 @@
 ---
 layout: default
-title: Non-Go Prototyping
+title: Package development
 pre: "<b>5. </b>"
 weight: 5
-aliases:
-  - /prototyping.html
+chapter: true
 ---
 
-To realize the full benefits of gokrazy, you need to use only
-software written in Go. If there is no Go software for what you want
-to do, creating that piece of software can pose a seemingly
-unsurmountable hurdle. To make some quick progress and figure out if
-your idea can be implemented, it might make sense to temporarily use
-existing software before starting your own implementation.
+# Package development
 
-This article shows a couple of techniques for getting non-Go
-software to work on gokrazy, in increasing order of complexity.
+A gokrazy package is simply a package of a Go software.
```

Comment:
We can make this a bit more tangible. How about:

A gokrazy package corresponds to a Go module. The `gokr-packer` program is a wrapper around `go install`.

Reply:
> A gokrazy package corresponds to a Go module. 

I think a Go module can have multiple packages, so this is also not optimal.

For instance the `github.com/VictoriaMetrics/VictoriaMetrics` module as many packages: https://github.com/VictoriaMetrics/VictoriaMetrics/tree/master/app

Reply:
Right, let’s make it:

We can make this a bit more tangible. How about:

A gokrazy package corresponds to a Go package (which in turn is part of a Go module). The `gokr-packer` program is a wrapper around `go install`.